src/components/signUp/Form.tsx

```diff
@@ -0,0 +1,128 @@
+import axios, { AxiosError } from 'axios';
+import { useState } from 'react';
+import 'components/signUp/Form.css'
+
+const Form = () => {
+  const [email, setEmail] = useState("");
+  const [password, setPassword] = useState("");
+  const [passwordConfirmation, setpasswordConfirmation] = useState("");
+  const [name, setName] = useState("");
+  const [gender, setGender] = useState("");
+  const [requestError, setrequestError] = useState(false);
+
+  async function SignUp() {
+    debugger;
+    try {
+      const res = await axios.post(
+        'http://localhost:3001/api/v1/users',
+        { user:
+          {
+            email: email,
+            password: password,
+            password_confirmation: passwordConfirmation,
+            gender: gender,
+            username: name
+          }
```

Comment:
Since this properties have the same name and value, you can do something like this:
```suggestion
          {
            email,
            password,
            password_confirmation: passwordConfirmation,
            gender,
            username: name
          }
```
WDYT?